bread/layout/skeleton.py

```diff
@@ -39,7 +39,11 @@ def default_page_layout(menu, *content):
             ShellHeader(
                 hg.C("PLATFORMNAME"),
                 hg.C("COMPANYNAME"),
-                searchbar=hg.F(lambda c: c.get("SEARCHBAR", "")),
+                searchbar=hg.If(
+                    hg.C("request.user.is_authenticated"),
+                    hg.C("SEARCHBAR", ""),
```

Comment:
```hg.C``` does only support a single argument... Did this not raise an exception? (If the value does not exists ```None``` is returned which is rendered as an empty string (i. e. not rendered).

Reply:
Thinking about it: Adding support for a default value could actually make sense in some cases...

Reply:
> `hg.C` does only support a single argument... Did this not raise an exception? (If the value does not exists `None` is returned which is rendered as an empty string (i. e. not rendered).

fixed it, thanks.